deploy/crds/wmc.openshift.io_v1alpha1_windowsmachineconfig_cr.yaml

```diff
@@ -1,7 +1,11 @@
 apiVersion: wmc.openshift.io/v1alpha1
 kind: WindowsMachineConfig
 metadata:
-  name: example-windowsmachineconfig
+  name: instance
 spec:
   # Add fields here
-  size: 3
+  replicas: 1
+  instanceType: "m4.large"
+  aws:
+    credentialAccountId: "default"
+    sshKeyPair: "abc"
```

Comment:
How about we make this `openshift-dev` for now given this what we will be using to test if we want to use the CR?

Reply:
I am fine with it.